### ISSUE TITLE
Validate other versions of `CITATION.cff` files

### ISSUE THREAD

**Issue by fwkoch** [curvenote/citation-cff-utils#2]
Currently this only validates 1.2.0 against the latest json schema. Older versions define yaml schema (e.g. https://github.com/citation-file-format/schema/blob/1.0.3/CFF-Core/schema.yaml); supporting these would require a separate _yaml_ schema validation library. I don't know to what extent older versions are used vs. 1.2.0, so we should assess that before putting in too much work on this.

**Comment by nichtich** [comment]
I don't think this is relevant, I'd better close the issue and focus on other features.

**Comment by rowanc1** [comment]
Thanks for the input, agreed!
